bindings/ios/MEGASdk.mm

```diff
@@ -1057,6 +1057,14 @@ - (void)setNodeFavourite:(MEGANode *)node favourite:(BOOL)favourite {
     self.megaApi->setNodeFavourite(node.getCPtr, favourite);
 }
 
+- (void)favouritesForParent:(nullable MEGANode *)node count:(nullable NSNumber *)count delegate:(id<MEGARequestDelegate>)delegate {
+    self.megaApi->getFavourites(node ? node.getCPtr : NULL, count ? count.intValue : NULL, [self createDelegateMEGARequestListener:delegate singleListener:YES]);
```

Comment:
don't need check if node is nil, use node.getCPtr

Change this: node ? node.getCPtr : NULL with node.getCPtr,

Use NSInteger and cast it.

repalce count ? count.intValue : NULL with (int)count